<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rubium</class>
 <widget class="QMainWindow" name="Rubium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rubium</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
font:20px &quot;Lucida Console&quot;;</string>
    </property>
    <property name="text">
     <string>Untitled-Rubium</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>1021</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(140, 255, 78);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1021</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1001</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 255, 127);
;border-width:0;
border-style:outset;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Open(&amp;O)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Close(&amp;C)</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>279</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Go to</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Top</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bottom</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>96</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
    <property name="displayIntegerBase">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Refresh(&amp;R)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Save(&amp;S)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>More Functions(&amp;X)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>261</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 170, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 125, 92);
color: rgb(255, 255, 255);
font:20px &quot;Lucida Console&quot;;</string>
    </property>
    <property name="text">
     <string>More</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>261</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Read Whole Text</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>View FullScreen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Find(&amp;D)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Copy All(&amp;A)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Find And Replace</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 170, 127);
border-radius:0px;
border-style:outset;</string>
     </property>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Español</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中文</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 200, 127);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Stop Reading</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(85, 170, 0);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>Save As(&amp;N)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(64, 182, 182);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(67, 154, 111);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Extra Options(&amp;O)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>130</y>
      <width>191</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(64, 182, 182);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(67, 154, 111);
font:20px&quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Underline</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(67, 154, 111);
font:20px&quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Italics</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(67, 154, 111);
font:20px&quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(67, 154, 111);
font:20px&quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Apply To All</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(67, 154, 111);
font:20px&quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>To Selected</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(67, 154, 111);
font:20px&quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Read Only</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(67, 154, 111);
color: rgb(255, 255, 255);
border-radius:0px;
}
QPushButton:hover{
	background-color: rgb(170, 170, 255);
}</string>
     </property>
     <property name="text">
      <string>Save As Rich Text</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
